ddl/reorg.go

```diff
@@ -176,8 +180,9 @@ func (w *worker) runReorgJob(t *meta.Meta, reorgInfo *reorgInfo, tblInfo *model.
 		job.SetRowCount(rowCount)
 		updateAddIndexProgress(w, tblInfo, rowCount)
```

Comment:
I think `len(reorgInfo.Elements)` should be pass into this function to make progress calculate correctly.

Reply:
Yes. But I think it's best to put this in another PR. Some operation results and statistical information need to be modified. I'll create an issue first.

Reply:
Sure, good idea.

Reply:
Related to #19923.